model/lifecycle.go

```diff
@@ -31,6 +31,23 @@ const (
 	AllStatuses     = "*"
 )
 
+type RestartOptions struct {
```

Comment:
I'm not sure if this is the right place for these structs. Used to be in task_lifecycle, but is now also used in patch_lifecycle.